### ISSUE TITLE
LinkState(s), ModelState(s), etc should include a timestamp

### ISSUE THREAD

**Issue by anuppari** [ros-simulation/gazebo_ros_pkgs#535]
These messages should include a timestamp corresponding to the sim time since there is a nonzero delay between the time corresponding to state values and the time a callback on a topic e.g., `/gazebo/link_states` gets called.

**Comment by j-rivero** [comment]
Agree. We would need a volunteer to implement the feature and send the PR.
